Car.js

```diff
@@ -1,28 +1,80 @@
-function Car(make, model, year, color, seats){
+function Car(make, model, year, color, seats, passengers){
   this.make = make;
   this.model = model;
   this.year = year;
   //TODO: add color, seats here
-
+  this.color = color;
+  this.seats = seats;
   this.running = false;
   this.owner = "manufacturer";
   this.previousOwners = [];
+  this.passengers = passengers || [];
 }
 
 Car.prototype.sell = function(newOwner){
-
+	this.previousOwners.push(this.owner);
+	this.owner = newOwner;
 };
 
 Car.prototype.paint = function(newColor){
+	this.color = newColor;
+};
+
+Car.prototype.start = function() {
+	this.running = true;
+};
 
+Car.prototype.off = function() {
+	this.running = false;
 };
 
+Car.prototype.driveTo = function(destination) {
+	if (this.running) {
+		console.log("driving to " + destination);
+		return true;
+	}
+	return false;
+};
 
+Car.prototype.park = function() {
+	if (!this.running) {
+		console.log("parked!!");
+		return true;
+	}
+	return false;
+};
 
 
 
+Car.prototype.pickUp = function(name) {
+	if (this.running) {
+		if (this.seats - 1 > this.passengers.length) { 
```

Comment:
You could also use an && here in your conditional so that you don't need a nested if statement:
```if (this.running && this.passengers.length < this.seats - 1) {```